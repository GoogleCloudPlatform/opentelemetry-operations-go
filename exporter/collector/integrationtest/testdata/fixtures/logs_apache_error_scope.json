{
  "resourceLogs": [
    {
      "resource": {
        "attributes": [
          {
            "key": "cloud.platform",
            "value": {
              "stringValue": "gcp_compute_engine"
            }
          }
        ]
      },
      "scopeLogs": [
        {
          "scope": {
            "name": "scopeNameFoo",
            "version": "9000"
          },
          "logRecords": [
            {
              "timeUnixNano": "1651013315606298000",
              "body": {
                "kvlistValue": {
                  "values": [
                    {
                      "key": "time",
                      "value": {
                        "stringValue": "Tue Apr 26 22:48:35.606298 2022"
                      }
                    },
                    {
                      "key": "severity",
                      "value": {
                        "stringValue": "notice"
                      }
                    },
                    {
                      "key": "message",
                      "value": {
                        "stringValue": "[pid 769:tid 140159306111872] AH00094: Command line: '/usr/local/apache/bin/httpd'"
                      }
                    }
                  ]
                }
              },
              "attributes": [
                {
                  "key": "log.file.name",
                  "value": {
                    "stringValue": "test.log"
                  }
                },
                {
                  "key": "gcp.log_name",
                  "value": {
                    "stringValue": "apache-error-fixture"
                  }
                }
              ],
              "traceId": "",
              "spanId": ""
            }
          ]
        }
      ]
    }
  ]
}
