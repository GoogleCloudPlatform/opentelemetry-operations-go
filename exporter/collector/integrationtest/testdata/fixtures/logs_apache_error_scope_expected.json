{
  "writeLogEntriesRequests": [
    {
      "entries": [
        {
          "logName": "projects/fakeprojectid/logs/apache-error-fixture",
          "resource": {
            "type": "gce_instance",
            "labels": {
              "instance_id": "",
              "zone": ""
            }
          },
          "jsonPayload": {
            "message": "[pid 769:tid 140159306111872] AH00094: Command line: '/usr/local/apache/bin/httpd'",
            "severity": "notice",
            "time": "Tue Apr 26 22:48:35.606298 2022"
          },
          "timestamp": "1970-01-01T00:00:00Z",
          "labels": {
            "instrumentation_source": "scopeNameFoo",
            "instrumentation_version": "9000",
            "log.file.name": "test.log"
          }
        }
      ],
      "partialSuccess": true
    }
  ]
}
